pfcpiface/parse-far.go

```diff
@@ -141,7 +148,26 @@ func (f *far) parseFAR(farIE *ie.IE, fseid uint64, upf *upf, op operation) error
 			} else if f.dstIntf == ie.DstInterfaceCore {
 				f.tunnelIP4Src = ip2int(upf.coreIP)
 			}
+		case ie.PFCPSMReqFlags:
+			fields = Set(fields, FwdIEPfcpSMReqFlags)
+			smReqFlags, err := fwdIE.PFCPSMReqFlags()
+			if err != nil {
+				log.Println("Unable to parse PFCPSMReqFlags!")
+				continue
+			}
+			f.pfcpSMReqFlags = smReqFlags
+		}
+	}
+
+	if fields&FwdIEPfcpSMReqFlags != 0 {
+		if has2ndBit(f.pfcpSMReqFlags) {
+			f.sendEndMarker = true
+		} else {
+			f.sendEndMarker = false
 		}
+	} else {
+		f.pfcpSMReqFlags = 0
+		f.sendEndMarker = false
```

Comment:
can we move this into the switch case itself? Will this allow to get rid of `pfcpSMReqFlags`

Reply:
done